test/e2e/utils.go

```diff
@@ -79,6 +79,9 @@ func GetWithRetry(sleepInterval time.Duration, retryCount int, retryFunc func()
 	var err error
 	for i := 0; i < retryCount; i++ {
 		err = retryFunc()
+		if err == nil {
```

Comment:
@realshuting, I am a bit confused if we will need this anymore because of the else statement? 
https://github.com/kyverno/kyverno/blob/4765d1ccc9b8c691aba756151beb6ab026e1599b/test/e2e/utils.go#L88

Reply:
We can remove it, it's a duplicate of the `else` block.